embeddings/embedding/sklearn_embedding.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Dict, Optional, Union
+
+import pandas as pd
+from numpy import typing as nptyping
+from sklearn.base import BaseEstimator as AnySklearnVectorizer
```

Comment:
I'd leave BaseEstimator here since we do not bound it to any inherited class and it may be redundant (argument name says it should be vectorizer)

Reply:
I see the conception now with classifiers etc. so just discord this comment please